digiroad2-oracle/src/test/scala/fi/liikennevirasto/digiroad2/util/DataFixture.scala

```diff
@@ -703,6 +703,95 @@ object DataFixture {
   }
 
 
+  def fillLaneAmountsMissingInRoadLink(): Unit = {
+    println("\nFill Lane Amounts in missing road links")
+    println(DateTime.now())
+
+    val LanesNumberAssetTypeId = 140
+    val NumberOfRoadLanesMotorway = 2
+    val NumberOfRoadLanesSingleCarriageway = 1
+
+    //Get All Municipalities
+    val municipalities: Seq[Int] =
+    OracleDatabase.withDynSession {
+      Queries.getMunicipalities
+    }
+
+    println("Obtaining all Road Links By Municipality")
+
+    //For each municipality get all VVH Roadlinks for pick link id and pavement data
+    municipalities.foreach { municipality =>
+
+      println("Start processing municipality %d".format(municipality))
+
+      //Obtain all RoadLink by municipality
+      val roadLinks = roadLinkService.getRoadLinksFromVVHByMunicipality(municipality)
```

Comment:
We thought that use cache data in batch process it isn't a good method. But I will change it to the method suggested by Ossi